py/examples/scripts/ChicagoCrimeDemo.py

```diff
@@ -222,12 +215,12 @@ def score_event(crime, model, censusTable):
     probOfArrest = predictTable["true"][0,0]
     return probOfArrest
 
-for crime in crime_examples:
-    arrestProbGBM = 100*score_event(crime, gbm_model, df_census)
-    arrestProbDLM = 100*score_event(crime, dl_model, df_census)
+for i in crime_examples:
```

Comment:
Not so important, but 'crime' seems explaining what is the object we are iterating over better than 'i'. Or we can use anything else for the name